### PR TITLE
[PM-5537] Remove Unecessary Biometric State

### PR DESCRIPTION
**Pull request by MGibson1**
We don't need to determine platform in desktop background, it can be done in the UI at any time.

## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [x] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Removes unused or things that can be retrieved at runtime easily from biometric state.

### Draft Reason
Depends on #7761 and will be retargeted after merging.

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: apps/desktop/src/utils.ts [hunk 1]
```diff
@@ -25,6 +25,17 @@ export function isDev() {
   return process.defaultApp || /node_modules[\\/]electron[\\/]/.test(process.execPath);
 }
 
+export function getPlatform() {
```
Comment (jlf0dev) [inline apps/desktop/src/utils.ts:28]: Question: why this instead of using `ElectronPlatformUtilsService`?

**Review by jlf0dev — COMMENTED** [review]
Just a small question. Nice work on cleaning up the biometrics text, I like this way a LOT more

File: apps/desktop/src/auth/lock.component.ts [hunk 2]
```diff
@@ -183,4 +184,15 @@ export class LockComponent extends BaseLockComponent {
...
+  get biometricText() {
+    switch (getPlatform()) {
+      case "mac":
+        return "unlockWithTouchId";
+      case "windows":
+        return "unlockWithWindowsHello";
+      default:
+        throw new Error("Unsupported platform");
+    }
+  }
```
Comment (Hinton) [inline apps/desktop/src/auth/lock.component.ts:197]: This is duplicated in `settings.component`, do we want this to be refactored or do we expect the texts to diverge in the future.

File: apps/desktop/src/utils.ts (hunk 1 above)
Comment (MGibson1) [reply apps/desktop/src/utils.ts:28]: I can put it in there, but I don't see a method on there that gives you the OS you're on. Am I missing it?

File: apps/desktop/src/auth/lock.component.ts (hunk 2 above)
Comment (MGibson1) [reply apps/desktop/src/auth/lock.component.ts:197]: I didn't want to assume we'd keep them the same and it's not much repeated code so I didn't really think about where to put it. Do you have a suggestion?

File: apps/desktop/src/utils.ts (hunk 1 above)
Comment (jlf0dev) [reply apps/desktop/src/utils.ts:28]: I believe `getDevice()` would give you the `DeviceType`

File: apps/desktop/src/auth/lock.component.ts (hunk 2 above)
Comment (Hinton) [reply apps/desktop/src/auth/lock.component.ts:197]: PlatformUtilsService 😭. It already handles calling biometric. Or lift out all the logic to a BiometricService

**Review by jlf0dev — APPROVED** [review]

**Review by Hinton — APPROVED** [review]

**Pull request merged**